### PR TITLE
Security

### PR DESCRIPTION
**Pull request by PerLizard**

### CODE REVIEW

File: src/main/resources/test/data-test.sql
Comment (Neutralas) [inline src/main/resources/test/data-test.sql:1]: move content into test-data.sql file

**Review by WilliamMoolman — CHANGES_REQUESTED** [review]
The tests other than the security tests fail as they are not authenticating requests. In those tests, can you replace `@AutoConfigureMockMvc` with `@AutoConfigureMockMvc(addFilters = false)` to disable authentication?

**Review by WilliamMoolman — APPROVED** [review]
Really really nice stuff

**Review by Neutralas — APPROVED** [review]
thanks! great work!

**Pull request merged**